release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -184,6 +184,18 @@ With this update, you can customize workload notifications on the *User Preferen
 ==== Improved quota visibility
 With this update, non-admin users are now able to view their usage of the `AppliedClusterResourceQuota` on the *Project Overview*, *ResourceQuotas*, and *API Explorer* pages in order to determine the cluster-scoped quota available for use. Additionally, `AppliedClusterResourceQuota` details can now be found on the *Search* page.
 
+[id="listing-pods-targeted-network-policies"]
+==== Listing pods targeted by network policies
+When using the network policy functionality in the {product-title} web console, the pods that are affected by a policy are listed. The list updates as the combined namespace and pod selectors in the following policy sections are modified:
+* Peer definition
+* Rule definition
+* Ingress
+* Egress
+
+[id="switching-between-yaml-and-web-console-form"]
+==== Switching between YAML and a web console form
+With this update, you can now switch between YAML and a web console form in the web console while saving any changes made. 
```

Comment:
```suggestion
With this update, you can now switch between YAML and a web console form in the {product-title} web console while saving any changes made. 
```
I stumbled over the last part of the sentence. Are you saying that changes are saved automatically when you switch between YAML a web console form?

Reply:
yes, updated. Thanks!